### ISSUE TITLE
Add dynamic link to track visitors to app from SM Classic

### ISSUE THREAD

**Issue by janieTC-USDS** [department-of-veterans-affairs/va-mobile-app#9150]
### 1.) Requestor's Name

Janie Tankard Carnock

### 2.) Requestor's Email

jane.carnock@va.gov

### 3.) VA Team

Health Products - Messages

### 4.) What is the name of the project or change?

Dynamic link for SM Classic banner

### 5.) Please explain what problem you are trying to solve.

- As of late June,  we now have a smart app banner implemented for mobile iOS users navigating to SM Classic on MHV. Per a request from Charles (VA Chief Technology Officer), our goal is to drive more traffic to the native mobile experience for users on a mobile device. 
- This was our MVP solution while we work on implementing for Android as well.
- Right now, we don't have a clear way of tracking how many users are coming in through this link. Adding a dynamic link for this banner would allow us to track how many users are using this link to adopt native VAHB mobile.

### 6.) Please explain how this change would work.

As per a July 2024 call between OCTO, VAHB mobile, and SM MHV Classic team members, we determined it should be feasible to deep link into SM. 

To execute, we noted the following:
- Need to make some changes to make browser deep link
- Would be extending what has been done for push notifications
- Example of tag: meta name="apple-itunes-app" content="app-id=1559609596"
- There is a certain parameter that would need to be added as a value to that attribute
- VAHB would provide that
- Noting that the user will still have to sign on and go through log in. However, with launch of new home-screen at end of July that elevates the unread messages link, i.e. a user will experience 1 click versus 3 in the current state. This is an acceptable UX. 

End result:
- We are able to assess what volume of users are adopting VAHB through this new banner.

### 7.) Features Impacted

Secure messaging

### 8.) Please include all relevant documentation links for this change request.

![image (14)](https://github.com/user-attachments/assets/25adedc9-9b1f-4345-bea2-788682c9f95c)

We need to add a link to the Open or View button that appears in the banner here.

### 9.) Does your team have all the necessary materials (documents, designs, processes, analytics, APIs, etc) in place for Mobile to make these changes right now?

yes

### 10.) Has this change been discussed with VAHB POs (Rachel, Ryan, or Don)?

Yes

### 11.) Has this change been agreed upon by all impacted VA Stakeholders?

Yes

### 12.) What is your ideal due date for this change request?

Within next quarter

**Comment by timwright12** [comment]
@janieTC-USDS Is the web team aware of this change? Since the link lives within va.gov, it will have to happen over there and the mobile team can support as needed

**Comment by janieTC-USDS** [comment]
Yes, Web team is aware aka SM MHV Classic team. From ticket above:

As per a July 2024 call between OCTO, VAHB mobile, and SM MHV Classic team members, we determined it should be feasible to deep link into SM.

To execute, we noted the following:

Need to make some changes to make browser deep link
**Would be extending what has been done for push notifications**
Example of tag: meta name="apple-itunes-app" content="app-id=1559609596"
There is a certain parameter that would need to be added as a value to that attribute
**VAHB would provide that**

**Comment by timwright12** [comment]
Gotcha, will pass this along. Thanks! cc @ajsarkar28 Global work, correct?

**Comment by BeccaTupaj** [comment]
@janieTC-USDS Are you able to provide an updated on the status and need for this work? I want to make sure I am understanding the current state of this request before I add it to our Mobile Team backlog.

**Comment by rtwell** [comment]
fysa I think Janie is no longer with VA/USDS.

**Comment by rtwell** [comment]
@kfink24 Do we still need to do this at this point? cc @klawyer

**Comment by kfink24** [comment]
Nope I would de-prioritize this since we will be re-directing Classic users to Va.gov in June.  But it may worth exploring for VA.gov users at a future time.  Still wouldn't be priority right now.  cc: @batemapf for SM product

**Comment by BeccaTupaj** [comment]
@rtwell given Kaitlin's response above can we close this change request?

**Comment by rtwell** [comment]
I will press my favorite button.
